### PR TITLE
Сообщаю об опечатках (прочитал первые 50 страниц)

### PR DESCRIPTION
**Pull request by banditos13**
Sorry, please remove wrong request

### CODE REVIEW

File: Initialization/linux-initialization-8.md [hunk 1]
```diff
@@ -305,13 +305,42 @@ The `bit_wait_table` is array of wait queues that will be used for wait/wake up
...
+
+Now let's get back to the our code and look on the two configuration options: `CONFIG_FAIR_GROUP_SCHED` and `CONFIG_RT_GROUP_SCHED`. The least unit which scheduler operates is an individual task or thread. But a process is not only one type of entities of which the scheduller may operate. Both of these options provides support for group scheduling. The first one option provides support for group scheduling with `completely fail scheduler` policies and the second with `real-time` policies respectively.
+
+In simple words, group scheduling is a feature that allows us to schedule a set of tasks as if a single task. For example, if you create a group with two tasks on the group, then this group is just like one normal task, from the kernel perspective. After a group is scheduled, the scheduler will pick a task from this group and it will be scheduled inside the group. So, such mechanism allows us to build hierarcies and manage their resources. Although a minimal unit of scheduling is a process, the Linux kernel scheduler does not use `task_struct` structure under the hood. There is special `sched_entity` strcture that is used by the Linux kernel scheduler as scheduling unit.
```
Comment (initBasti) [inline Initialization/linux-initialization-8.md]: s/In simple words, group scheduling is a feature that allows us to schedule a set of tasks **as if a single task.**/
   In simple words, group scheduling is a feature that allows us to schedule a set of tasks **as if they were a single task.**

File: Initialization/linux-initialization-8.md [hunk 2]
```diff
@@ -305,13 +305,42 @@ The `bit_wait_table` is array of wait queues that will be used for wait/wake up
...
+
+The `real-time` policies are also supported for the time-critical applications: `SCHED_FIFO` and `SCHED_RR`. If you've read something about the Linux kernel scheduler, you can know that it is modular. It means that it supports different algorithms to schedule different types of processes. Usually this modularity is called `scheduler classes`. These modules encapsulate scheduling policy details and are handled by the scheduler core without knowing too much about them. 
+
+Now let's get back to the our code and look on the two configuration options: `CONFIG_FAIR_GROUP_SCHED` and `CONFIG_RT_GROUP_SCHED`. The least unit which scheduler operates is an individual task or thread. But a process is not only one type of entities of which the scheduller may operate. Both of these options provides support for group scheduling. The first one option provides support for group scheduling with `completely fail scheduler` policies and the second with `real-time` policies respectively.
```
Comment (initBasti) [inline Initialization/linux-initialization-8.md]: Correct typo and English grammar.

s/The least unit which scheduler operates is an individual task or thread. But a process is not only one type of entities of which the scheduller may operate./
**The smallest unit on which the scheduler operates** is an individual task or thread. But a process is not **the only type of entity on which the scheduler** may operate.

Comment (initBasti) [inline Initialization/linux-initialization-8.md]: Correct typo, provide**s** is only used for he/she/it

s/Both of these options provides support for group scheduling/
   Both of these options **provide** support for group scheduling

Comment (initBasti) [inline Initialization/linux-initialization-8.md]: Remove redundant word and add 'the' at the appropriate places + correct typo s/fail scheduler/fair scheduler/

s/The first one option provides support for group scheduling with `completely fail scheduler` policies and the second with `real-time` policies respectively./
The **first option** provides support for group scheduling with **the** `completely fair scheduler` policies and the second with **the** `real-time` policies respectively.

File: Initialization/linux-initialization-8.md [hunk 3]
```diff
@@ -305,13 +305,42 @@ The `bit_wait_table` is array of wait queues that will be used for wait/wake up
...
+* `SCHED_NORMAL`;
+* `SCHED_BATCH`;
+* `SCHED_IDLE`.
+
+The `SCHED_NORMAL` is used for the most normal applications, the amount of cpu each process consumes is mostly determined by the [nice](http://en.wikipedia.org/wiki/Nice_%28Unix%29) value, the `SCHED_BATCH` used for the 100% non-interactive tasks and the `SCHED_IDLE` runs tasks only when the processor has no task to run besides this task.
+
+The `real-time` policies are also supported for the time-critical applications: `SCHED_FIFO` and `SCHED_RR`. If you've read something about the Linux kernel scheduler, you can know that it is modular. It means that it supports different algorithms to schedule different types of processes. Usually this modularity is called `scheduler classes`. These modules encapsulate scheduling policy details and are handled by the scheduler core without knowing too much about them.
```
Comment (initBasti) [inline Initialization/linux-initialization-8.md:321]: Change sentence beginning

s/It means that it supports different algorithms to schedule different types of processes./
**That means** it supports different algorithms to schedule different types of processes.

File: Initialization/linux-initialization-8.md [hunk 4]
```diff
@@ -305,13 +305,42 @@ The `bit_wait_table` is array of wait queues that will be used for wait/wake up
...
+
+In simple words, group scheduling is a feature that allows us to schedule a set of tasks as if a single task. For example, if you create a group with two tasks on the group, then this group is just like one normal task, from the kernel perspective. After a group is scheduled, the scheduler will pick a task from this group and it will be scheduled inside the group. So, such mechanism allows us to build hierarcies and manage their resources. Although a minimal unit of scheduling is a process, the Linux kernel scheduler does not use `task_struct` structure under the hood. There is special `sched_entity` strcture that is used by the Linux kernel scheduler as scheduling unit.
+
+So, the current goal is to calculate a space to allocate for a `sched_entity(ies)` of the root task group and we do it two times with:
```
Comment (initBasti) [inline Initialization/linux-initialization-8.md:327]: Improve wording

s/
So, the current goal is to calculate a space to allocate for a `sched_entity(ies)` of the root task group and we do it two times with:/
**The goal is to calculate, how much space we need to allocate for the `sched_entity(ies)` of the root task group:**

File: Interrupts/linux-interrupts-2.md [hunk 5]
```diff
@@ -224,7 +224,7 @@ ENTRY(early_idt_handler_array)
...
+It fills `early_idt_handler_array` with the `.rept NUM_EXCEPTION_VECTORS` and contains entry of the `early_make_pgtable` interrupt handler (more about its implementation you can read in the part about [Early interrupt and exception handling](https://0xax.gitbooks.io/linux-insides/content/Initialization/linux-initialization-2.html)). For now we come to the end of the `x86_64` architecture-specific code and the next part is the generic kernel code. Of course you already can know that we will return to the architecture-specific code in the `setup_arch` function and other places, but this is the end of the `x86_64` early code.
```
Comment (initBasti) [inline Interrupts/linux-interrupts-2.md:227]: Improve wording & link https://0xax.gitbooks.io/linux-insides/content/Initialization/linux-initialization-2.html seems to be dead

s/(more about its implementation you can read in the part about [Early interrupt and exception handling](https://0xax.gitbooks.io/linux-insides/content/Initialization/linux-initialization-2.html))/
(**you can read more about its implementation in the part about** [Early interrupt and exception handling](https://0xax.gitbooks.io/linux-insides/content/Initialization/linux-initialization-2.html))/

Comment (initBasti) [inline Interrupts/linux-interrupts-2.md:227]: Improve wording

s/For now we come to the end of the `x86_64` architecture-specific code/**For now, we have reached the end** of the `x86_64` architecture-specific code/

Comment (initBasti) [inline Interrupts/linux-interrupts-2.md:227]: Improve wording

s/Of course you already can know that we will return to the architecture-specific code/
**You probably already know,** that we will return to the architecture-specific code/

**Comment by 0xAX** [comment]
Hello @banditos13, I see that the pull request contains a weird mix of fixes for russian translation and english parts as well. Probably you selected a wrong repo during sending of the pull request?

**Pull request closed without merging**